src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,61 @@
  */
 
 public class Plane {
+    private String name;
+    private String type;
+    private int maxSpeed;
+    private int maxSeats;
+    private double maxCargo;
 
+    private Plane(PlaneBuilder builder) {
+        this.name = builder.name;
+        this.type = builder.type;
+        this.maxSpeed = builder.maxSpeed;
+        this.maxCargo = builder.maxCargo;
+        this.maxSeats = builder.maxSeats;
+    }
+
+    public static class PlaneBuilder {
+        private String name;
+        private String type;
+        private int maxSpeed;
+        private double maxCargo;
+        private int maxSeats;
+
+        public PlaneBuilder() {
+        }
```

Comment:
so, do we have to create public constructor with no parameters?)

Reply:
or we can create it like this Plane.PlaneBuilder?

Reply:
No, we don't need it. Sorry, I tried to do it another way and got confused. Solved.